bucket/dotnet-sdk.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "7.0.200",
+    "version": "8.0.100-preview.1.23115.2",
```

Comment:
Preview versions are for the _Versions_ bucket. We need to look only for the stable versions here.

Reply:
OK, I got it